modules/distr-tracing-tempo-config-storage.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-storage_{context}"]
+= Distributed tracing storage configuration options
+
+You configure storage for the Collector, Ingester, Distributor, and Query services under `spec.storage`. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
+
+.General storage parameters used by the {TempoOperator} to define distributed tracing storage
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|spec:
+  storage:
+    type:
+|Type of storage to use for the deployment.
+|`memory`. Memory storage is only appropriate for development, testing, demonstrations, and proof of concept environments because the data does not persist when the pod is shut down.
+|`memory`
+
+|storage:
+  secretname:
+|Name of the secret, for example `tracing-secret`.
+|
+|N/A
+
+|storage:
+  options: {}
+|Configuration options that define the storage.
+|
+|
+|===
```

Comment:
We need to add how to create secrets for each type. but here I have a quetion on which types are we goint to support

Anyway I added how to create secret for all of them based on comunity docummentation